Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/CommandLineOptions.cs

```diff
@@ -172,6 +181,7 @@ public static void PrintUsage()
             Console.WriteLine("\t--pause-exit <true or false>          If set to true the test tool will pause waiting for a key input before exiting. The is useful");
             Console.WriteLine("\t                                      when executing from an IDE so you can avoid having the output window immediately disappear after");
             Console.WriteLine("\t                                      executing the Lambda code. The default value is true.");
+            Console.WriteLine("\t--response-only                       Logs response only or any exceptions (only works when --no-ui is present).");
```

Comment:
I think `--disable-logs` is clearer what the PR is doing then `--response-only`.

Nit picking on the help wording change `only works when --no-ui is present` to `switch is valid when using --no-ui`. I feel like saying "only works" implies a limitation not a design choice.

Reply:
Done. Please review.